cg/meta/workflow/mutant/mutant.py

```diff
@@ -49,6 +55,10 @@ def get_case_path(self, case_id: str) -> Path:
     def get_case_output_path(self, case_id: str) -> Path:
         return Path(self.get_case_path(case_id=case_id), "results")
 
+    def get_case_results_file_path(self, case: Case) -> Path:
+        case_output_path: Path = self.get_case_output_path(case.internal_id)
+        return Path(case_output_path.as_posix, f"/sars-cov-2_{case.latest_ticket}_results.csv")
```

Comment:
There is no need to add `as_posix` as you can add a path as an argument to the Path constructor. ( I get a PyCharm warning from this)
```suggestion
        return Path(case_output_path, f"/sars-cov-2_{case.latest_ticket}_results.csv")
```